### PR TITLE
fix: add injection of lacking arguments of customPrettifiers

### LINKED ISSUE #500: PinoPretty.customPrettifiers() lacks arguments declared in type PinoPretty.Prettifier
Here is the type defenition:
Знімок екрана 2024-03-24 о 19 29 34

And here is JS code for PinoPretty.customPrettifiers(): 
Знімок екрана 2024-03-24 о 19 31 22

As you can see we are lacking of injection of all arguments from type defenition

### PR DESCRIPTION
**Pull request by pasha-vuiko**
resolves #500

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
Thanks for opening a PR! Can you please add a unit test?

**Comment by pasha-vuiko** [comment]
> Thanks for opening a PR! Can you please add a unit test?

done

**Review by mcollina — APPROVED** [review]
lgtm

File: lib/utils/prettify-metadata.test.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 'use strict'
 
+const getColorizer = require('../colors')
 const tap = require('tap')
 const prettifyMetadata = require('./prettify-metadata')
 const context = {
```
Comment (jsumners) [inline lib/utils/prettify-metadata.test.js:6]:
```suggestion
const tap = require('tap')
const prettifyMetadata = require('./prettify-metadata')
const getColorizer = require('../colors')
const context = {
```

**Review by jsumners — CHANGES_REQUESTED** [review]

File: lib/utils/prettify-metadata.test.js (hunk 1 above)
Comment (pasha-vuiko) [reply lib/utils/prettify-metadata.test.js]: fixed

**Comment by pasha-vuiko** [comment]
@jsumners could you re-review please?

**Comment by pasha-vuiko** [comment]
@jsumners hey 👋, what about re-review?🙂

**Review by jsumners — APPROVED** [review]

**Pull request merged**